transaction/core/src/onetime_keys.rs

```diff
@@ -3,17 +3,19 @@
 //! # MobileCoin transactions use CryptoNote-style `onetime keys` to protect
 //! recipient privacy.
 //!
-//! When creating a transaction, the sender computes a onetime public key for
-//! each output in such a way that only the sender and the recipient know who
-//! the recipient is, and only the recipient is able to compute the
-//! corresponding onetime private key that is required to spend the output.
+//! When creating a transaction, the sender computes a one-time use public key
+//! for each output in such a way that only the sender and the recipient know
+//! who the recipient is, and only the recipient is able to compute the
+//! corresponding private key that is required to spend the output.
 //!
 //! To further protect recipient privacy, an output's onetime key is computed
 //! for a `subaddress` that the recipient generated from their CryptoNote-style
 //! address. This makes it easy for a recipient to use different subaddresses
 //! for different purposes and keep track of how much MobileCoin was sent to
 //! each subaddress.
 //!
+//! This key is called the "target key" of the TxOut.
```

Comment:
I guess that this is actually not the right thing to say:
the comment below about target key being the "onetime public key" is what was correct

So maybe we should just say, the public key is called "target key" of the TxOut, and the corresponding private key is the "onetime private key"? Since I think this commit isn't actually getting rid of the term `onetime_private_key` in source code